docroot/modules/contrib/videojs/videojs.libraries.yml

```diff
@@ -9,4 +9,29 @@ videojs:
     //vjs.zencdn.net/5.0/video.min.js: { type: external, minified: true }
   css:
     component:
-      //vjs.zencdn.net/5.0/video-js.min.css: { type: external, minified: true }
\ No newline at end of file
+      //vjs.zencdn.net/5.0/video-js.min.css: { type: external, minified: true }
+
+videojs_annotation:
+  version: VERSION
+  license:
+    name: Apache 2.0
+    url: https://github.com/videojs/video.js/blob/master/LICENSE
+    gpl-compatible: true
+  js:
+    js/jquery.min.js: { minified: true }
+    js/annotator-full.js: {}
+    js/video.min.js: { minified: true }
+    js/vjs.youtube.js: {}
+    js/rangeslider.min.js: { minified: true }
+    js/share-annotator.min.js: { minified: true }
+    js/ova.min.js: { minified: true }
+    js/addannotatebutton.js: {}
+    js/jquery-noconflict.js: {}
+  css:
+    theme:
+      //assets.annotateit.org/annotator/v1.2.5/annotator.min.css: { minified: true }
```

Comment:
Do we need these? If not, lets remove the commented out code or comment with the reason for keeping it here.